dags/sql/select-sensor_id_count_lookback.sql

```diff
@@ -31,7 +45,13 @@ SELECT
     'Pass threshold: ' || to_char(
             FLOOR({{ params.threshold }}::numeric * AVG(lb.lookback_count)),
```

Comment:
so technically the passing value could change overtime ?

Reply:
Yeah! I think it makes sense, if we add/remove detectors. As long as there's no sudden downwards spike.